plugins/chat/assets/javascripts/discourse/modifiers/chat/did-mutate-childlist.js

```diff
@@ -1,24 +0,0 @@
-import Modifier from "ember-modifier";
```

Comment:
This is not needed anymore and was buggy anyways, it would for example trigger when you hover a message